doc/release_instructions.md

```diff
@@ -126,16 +118,30 @@ git push
 
 ### Release! [apply step only when running point-release or release-candidate target]
 
-* To release a major, minor, or patch version, run `make point-release`
-* To release an alpha version, run e.g.
-  `make VERSION_ADDITIONAL=-alpha1 release-candidate`
-* To release a beta version, run e.g.
-  `make VERSION_ADDITIONAL=-beta1 release-candidate`
-* To release an rc, run e.g. `make VERSION_ADDITIONAL=-rc1 release-candidate`
-
-Then update the
-[release's notes](https://github.com/pachyderm/pachyderm/releases), and post
-the update on the #users channel.
+* To release a major, minor, or patch version, run
+```shell
+make point-release
+```
+* To release an alpha/beta/RC version, specify the additional text to appear in the release version and run
+```shell
+ make VERSION_ADDITIONAL=-alpha1 release-candidate
```

Comment:
Nit: the lines in this code block (`make`, `OR`, etc) seem like they have leading spaces?